fusionauth/resource_fusionauth_tenant_test.go

```diff
@@ -637,7 +640,7 @@ resource "fusionauth_tenant" "test_%[1]s" {
       #notify_user_email_template_id = "UUID"
       on_login                      = "NotifyUser"
     }
-    max_length = 100
+    max_length = 50
```

Comment:
Is this change for testing only, or is it required to complete a test?

Reply:
This was due to the new constraint on bcrypt password lengths. The example code and test configurations were no longer valid.

Reply:
Ah, got it. Thanks, didn't see we were setting this to `bcrypt`.